plugins/scaffolder-actions/scaffolder-backend-argocd/package.json

```diff
@@ -33,11 +33,10 @@
     "clean": "backstage-cli clean"
   },
   "dependencies": {
-    "@backstage/backend-common": "^0.24.0",
     "@backstage/config": "^1.2.0",
-    "@backstage/plugin-scaffolder-backend": "^1.24.0",
-    "@roadiehq/backstage-plugin-argo-cd-backend": "^3.0.4",
-    "@backstage/backend-test-utils": "^0.5.0",
+    "@backstage/plugin-scaffolder-node": "^0.4.10",
+    "@backstage/plugin-scaffolder-node-test-utils": "^0.1.11",
+    "@roadiehq/backstage-plugin-argo-cd-backend": "^3.0.2",
     "winston": "^3.2.1"
```

Comment:
You removed the only reference to `Winston` below. Can't this be cleaned up now?

Reply:
let me do that.